### PR TITLE
Add descriptions to connection_connect_timeout

### PR DESCRIPTION
**Pull request by YihaoOct**
Updated documentation on `connection_connect_timeout` to specify that the value is in nanoseconds and that the min value has to be 10 seconds.

This PR also included documentation updates from previous merged PRs.

### CODE REVIEW

File: docs/data-sources/azure_cloud_service_deployment_targets.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ Read-Only:
 - `client_certificate_variable` (String)
 - `cloud_service_name` (String)
 - `cluster_certificate` (String)
+- `cluster_certificate_path` (String)
```
Comment (YihaoOct) [inline docs/data-sources/azure_cloud_service_deployment_targets.md:101]: Most changes in this PR are from a re-run of `make docs`. Looks like previous several merged PRs did not include the documentation updates. So these are picked up in this PR.

File: octopusdeploy/schema_machine_policy.go [hunk 2]
```diff
@@ -129,9 +129,10 @@ func getMachinePolicyDataSchema() map[string]*schema.Schema {
 func getMachinePolicySchema() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		"connection_connect_timeout": {
-			Default:  time.Minute,
-			Optional: true,
-			Type:     schema.TypeInt,
+			Default:     time.Minute,
+			Optional:    true,
+			Type:        schema.TypeInt,
+			Description: "In nanoseconds, no less than 10 seconds (10000000000)",
```
Comment (YihaoOct) [inline octopusdeploy/schema_machine_policy.go:135]: This is the main change. See: https://octopusdeploy.slack.com/archives/C03SG0LFJHX/p1692940267454359

Comment (APErebus) [reply octopusdeploy/schema_machine_policy.go:135]: I'd probably slightly adjust this to `In nanoseconds. Minimum value: 10000000000 (10 seconds).`

File: octopusdeploy/schema_machine_cleanup_policy.go [hunk 3]
```diff
@@ -55,9 +55,10 @@ func getMachineCleanupPolicySchema() map[string]*schema.Schema {
 			}, false)),
 		},
 		"delete_machines_elapsed_timespan": {
-			Computed: true,
-			Optional: true,
-			Type:     schema.TypeInt,
+			Computed:    true,
+			Optional:    true,
+			Type:        schema.TypeInt,
+			Description: "In nanoseconds.",
```
Comment (YihaoOct) [inline octopusdeploy/schema_machine_cleanup_policy.go:61]: I also added this description to all time-related fields that come in nanoseconds.

**Review by APErebus — APPROVED** [review]
Looks good mate 👍

**Review by johnsimons — APPROVED** [review]

**Pull request merged**